tests/fixtures/tests.py

```diff
@@ -778,3 +778,17 @@ def test_format_discovery(self):
             '<Article: Time to reform copyright>',
             '<Article: Poker has no place on ESPN>',
         ])
+
+
+class PostgresNulLoadingTests(TestCase):
+
+    @unittest.skipUnless(connection.vendor == 'postgresql', "PostgreSQL specific")
+    def test_loading_nul_0x00_chars(self):
+        apps.clear_cache()
+        Article.objects.all().delete()
+        Category.objects.all().delete()
+        Site.objects.all().delete()
+        # Load Postgres fixture
+        with self.assertRaises(ValueError) as cm:
+            management.call_command('loaddata', 'postgres_nul.json', verbosity=0)
+            self.assertIn('A string literal cannot contain NUL (0x00) characters.', cm.exception.args[0])
```

Comment:
This should also include the "Could not load .." part of the error message -- otherwise the test passes without the change.